sketch.js

```diff
@@ -18,11 +18,11 @@ let seperationOn = true;
 let boids = [];
 
 // Colors
-let RED = [250, 0, 0];
-let GREEN = [0, 255, 0];
-let BLUE = [0, 0, 255];
-let YELLOW = [255, 255, 0];
-let MAGENTA = [255, 0, 255];
+let RED = [100, 0, 0];
+let GREEN = [0, 100, 0];
+let BLUE = [0, 0, 1000];
+let YELLOW = [100, 100, 0];
```

Comment:
We don't need to change the color definations.

Reply:
ok i will chnge it